tests/test_header.py

```diff
@@ -298,7 +298,7 @@ def test_find_and_replace_keep_shebang():
         #!/usr/bin/env python3
 
         # spdx-FileCopyrightText: Jane Doe
-        # spdx-FileCopyrightText: Mary Sue
+        # spdx-FileCopyrightText: Jane Doe
```

Comment:
There are two Jane Does here :) One of them should probably be the kindly John Doe.

Reply:
Ok, I can fix that